Test/Management/Integration/AirQualityDataClientIntegrationTests.cs

```diff
@@ -35,15 +31,25 @@ public async Task GetRequest_ValidData_CanReceiveSuccessResponse()
 
             var airClient = new AirQualityDataClient(configMock.Object);
 
-            var result = await airClient.GetCityAirQualityDataAsync(City.Wrap("Los Angeles"), State.Wrap("California"), CountryCode.US, CancellationToken.None);
+            var exception = await Assert.ThrowsAsync<Exception>(() => airClient.GetCityAirQualityDataAsync(City.Wrap("ase"), State.Wrap("California"), CountryCode.US, CancellationToken.None));
 
-            Assert.Equal("success", result.Status);
+            var resultME = await airClient.CalculateScoreForStateAsync(State.Wrap("ME"), CountryCode.US, CancellationToken.None);
+            Assert.Equal("Maine", resultME.Item1.Value);
         }
 
+        /// <summary>
+        /// Representation of configuration section
+        /// </summary>
         public class Section : IConfigurationSection
         {
+            /// <summary>
+            /// Gets api key
+            /// </summary>
+            /// <param name="key">api key string.</param>
             public string this[string key] { get => "ecc93ce2-d18c-44a3-a414-adc270da84bd"; set => throw new NotImplementedException(); }
-
+            
+            //public string this[string key] { get => "9ac310f1-7fb7-4ced-8808-478fdbe6ed13"; set => throw new NotImplementedException(); }
+            
```

Comment:
Apparently, the air quality API that we are using also has a daily limit (I'm not sure how many exactly). Thus, I kept two API keys in here just in case we need more testings without failing caused by the daily limit.

Reply:
Ideally intergration should only run a few times - during CI, usually.
I will take a deeper look, and mark this skip for local. Integration tests with third party api should not be called a lot.

Reply:
Thanks!